src/main/haskell/matching-logic/test/Kore/MatchingLogic/ProofSystem/OnePlusOne/ProofSystem.hs

```diff
@@ -155,17 +155,17 @@ instance (IsSignature sig, Eq (Sort sig), Eq (Label sig), Eq var) =>
                   Left (Error [] "right hand term does not match phi[y/x]")
               | otherwise -> Left (Error [] "conclusion does not agree with arguments")
             _ -> Left (Error [] "malformed conclusion")
-      ForallRule var term1 term2 ->
+      ForallRule _ term1 term2 ->
```

Comment:
var shouldn't be unused here, it should be tested to be the same as `sVar` and `sVar1`. This file was probably copied before I fixed that bug in the other proof system file.

Reply:
Can I merge those changes?  If I add `var == sVar` to the guard, ghc complains about missing Eq constraints in the ToPattern class

Reply:
Oh, wrong suggestion. The missing checks are `var == var1, var1 == var2`. You can compare with the guards in `Kore.MatchingLogic.ProofSystem.Minimal`.